From: =?utf-8?q?Zbigniew_J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 19 Feb 2014 09:13:09 -0500
Subject: pam-module: fix warning about ignoring vtnr

Feb 19 22:15:01 server crond[26418]: pam_systemd(crond:session): Ignoring vtnr
0 for  which is not seat0
---
 src/login/pam-module.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/login/pam-module.c b/src/login/pam-module.c
index 7ebb5d2..aa09ec1 100644
--- a/src/login/pam-module.c
+++ b/src/login/pam-module.c
@@ -335,7 +335,7 @@ _public_ PAM_EXTERN int pam_sm_open_session(
                         get_seat_from_display(display, NULL, &vtnr);
         }
 
-        if (seat && !streq(seat, "seat0")) {
+        if (seat && !streq(seat, "seat0") && vtnr != 0) {
                 pam_syslog(handle, LOG_DEBUG,
                       "Ignoring vtnr %d for %s which is not seat0", vtnr, seat);
                 vtnr = 0;
